ingestors/kafka/src/main/scala/hydra/kafka/endpoints/BootstrapEndpoint.scala

```diff
@@ -40,41 +40,44 @@ class BootstrapEndpoint(override val system:ActorSystem) extends RouteSupport
 
```

Comment:
What is it that changed in this file? Was this just reformatting? Just want to make sure I understand what is going on here.

Reply:
Reformatting. This allows me to use the ingestAltRoute in V1 and V2 tests.

Reply:
Isn't this for bootstrapping rather than ingest?

Reply:
Whoops! Did not realize the change in files... This was reformatting. I did not originally see that we had a BootstrapV2 file. So some work was done in this file to accommodate the V2 endpoint as well. This could be reverted without issue.

Reply:
Sounds good 🙂